pos_printer_network/doc/index.rst

```diff
@@ -15,23 +15,38 @@ Installation
 Configuration
 =============
 
+* Go to ``Point of Sale >> Configuration >> Point of Sale``
+
+  * Open POS configuration form
+  * Click on ``[Edit]``
+  * Specify IP address for ``Hardware Proxy / PosBox``
+
+Order Printers
+--------------
+
 * Go to ``Point of Sale >> Configuration >> Order Printers``
 
   * Click on ``[Create]``
-  * Specify a name of new printer in the "Printer Name" field
+  * Specify a name of new printer in the **Printer Name** field
   * Specify the proxy server IP address of network printer
-  * Check the "Network Printer" box if this printer is a network printer
+  * Check the **[x] Network Printer** box if this printer is a network printer
   * Select Product Categories
   * Click on ``[Save]``
+  * Open POS configuration form
```

Comment:
I think we don't need to include it to Order Printers menu, i.e. we need to add new line * Go to ``Point of Sale >> Configuration >> Point of Sale`` between